serverlessworkflow/modules/ROOT/pages/cloud/quarkus/deploying-on-openshift.adoc

```diff
@@ -0,0 +1,218 @@
+knative-openshift-5c7f985d55-4zggl           1/1     Running   0          22d
+knative-openshift-ingress-864c78b45c-mshpr   1/1     Running   0          22d
+knative-operator-webhook-8675c49975-k6wvn    1/1     Running   0          22d
+----
+
+When everything is ok, you can go ahead and make sure that the Knative Serving is configured, it is sufficient for this example only. But for more complex examples, you must have the
+link:{ocp_knative_eventing_url}[Knative Eventing] configured as well.
+will be enough, but for more complex examples, you might be required to have the
+link:{ocp_knative_eventing_url}[Knative Eventing] configured as well.
+
+To check if `Knative Serving` is deployed, execute the following command:
+
+[source,shell]
+----
+oc get knativeserving.operator.knative.dev/knative-serving -n knative-serving --template='{{range .status.conditions}}{{printf "%s=%s\n" .type .status}}{{end}}'
+----
+
+The result should me similar to:
+[source,properties]
+----
+DependenciesInstalled=True
+DeploymentsAvailable=True
+InstallSucceeded=True
+Ready=True
+----
+
+If you got 'False' as a result for the items above, it means that `Serving` is not fully installed, to install, deploy
+the `KnativeServing` definition:
+
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+    name: knative-serving
+    namespace: knative-serving
+----
+
+Another way to check if it is running, is to query the Pods from the `knative-serving` namespace:
+
+[source,shell]
+----
+oc get pods  -n knative-serving
+NAME                                     READY   STATUS    RESTARTS   AGE
+activator-5bdfdbcf44-q7cjt               2/2     Running   0          23h
+activator-5bdfdbcf44-wct6w               2/2     Running   0          23h
+autoscaler-6dff958db8-rf6t6              2/2     Running   0          23h
+autoscaler-6dff958db8-zrmp4              2/2     Running   0          23h
+autoscaler-hpa-5db8575dbc-8nhhg          2/2     Running   0          23h
+autoscaler-hpa-5db8575dbc-h8lfq          2/2     Running   0          23h
+controller-7649d68db6-x9qnb              2/2     Running   0          23h
+controller-7649d68db6-z5q54              2/2     Running   0          23h
+domain-mapping-db4fcb48c-jzpqr           2/2     Running   0          23h
+domain-mapping-db4fcb48c-n8tj6           2/2     Running   0          23h
+domainmapping-webhook-6896bb6475-bvgf6   2/2     Running   0          23h
+domainmapping-webhook-6896bb6475-fqlm9   2/2     Running   0          23h
+webhook-85b79f6c95-q5sld                 2/2     Running   0          23h
+webhook-85b79f6c95-wfc54                 2/2     Running   0          23h
+----
+
+[IMPORTANT]
+====
+If you get error messages related to `Istio`, this link:{knative_istio_issue_url}[article] might be helpful.
+====
+
+[IMPORTANT]
+====
+From now on, we will be using the `kogito-serverless` namespace. To create execute:
+[source,shell]
+----
+oc new-project kogito-serverless
+----
+====
```

Comment:
Let's not have two of the same box following each other. I think the last information is unnecessary to be in a box. But we are removing this section anyway if you follow my other suggestion...